zanata-war/src/main/webapp/WEB-INF/layout/project/settings-tab-languages.xhtml

```diff
@@ -11,7 +11,7 @@
     #{msgs['jsf.Languages']}
   </h2>
 
-  <h:form id="settings-languages-form" styleClass="l--push-bottom-0">
```

Comment:
Should do the same for zanata-war/src/main/webapp/WEB-INF/layout/version/settings-tab-languages.xhtml